DESCRIPTION

```diff
@@ -1,23 +1,23 @@
 Type: Book
 Package: biomarker.catalog
 Title: Biomarker Analysis Catalog
-Version: 0.1.9.9000
-Date: 2023-05-09
+Version: 0.2.0
+Date: 2023-06-25
 Authors@R: c(
     person("Daniel", "Sabanes Bove", , "daniel.sabanes_bove@roche.com", role = c("aut", "cre")),
     person("Paolo", "Moroni", , role = "aut"),
     person("F. Hoffmann-La Roche AG", role = c("cph", "fnd"))
```

Comment:
Dinakar I think you should add yourself as author given your significant work on this